### PR TITLE
Use datatable in users index

### PR DESCRIPTION
**Pull request by nicolas-brousse**

### CODE REVIEW

File: app/components/list/data_table_component.rb [hunk 1]
```diff
@@ -59,8 +59,8 @@ def link_to_sort(label, attribute)
       current_direction = current_attribute == attribute ? params[:direction].to_sym : nil
 
       parameters = case current_direction
-                   when nil then { column: attribute, direction: :asc }
-                   when :asc then { column: attribute, direction: :desc }
+                   when nil then { column: attribute, direction: :desc }
+                   when :desc then { column: attribute, direction: :asc }
```
Comment (B-Rass) [inline app/components/list/data_table_component.rb:63]:
```suggestion
                   when nil then { column: attribute, direction: :asc }
                   when :asc then { column: attribute, direction: :desc }
```

Comment (B-Rass) [reply app/components/list/data_table_component.rb:63]: In fact, keep it like it was, it should order a to z first.

**Review by B-Rass — CHANGES_REQUESTED** [review]
My bad about the asc / desc changes

File: app/components/list/data_table_component.rb (hunk 1 above)
Comment (nicolas-brousse) [reply app/components/list/data_table_component.rb]: @B-Rass I changed it back

**Review by B-Rass — APPROVED** [review]

**Pull request merged**